conda/models/package_info.py

```diff
@@ -15,7 +19,30 @@
                                   'index_json_record', 'icondata', 'noarch'))
 
 
-class PackageInfo(Entity):
+class NoarchInfo(Entity):
+    type = StringField()
+    entry_points = ListField(string_types, required=False)
+
+
+class FileType(Enum):
+    hardlink = 1
+    softlink = 2
+    directory = 4
+
 
-    file = ListField(string_types)
-    # TODO: finish this
+class PathInfo(Entity):
+    path = StringField()
+    sha256 = StringField()
+    size_in_bytes = IntegerField()
+    file_type = EnumField(FileType)
```

Comment:
@kalefranz  should this be https://github.com/conda/conda/blob/master/conda/base/constants.py#L63 instead?